net/dns/manager.go

```diff
@@ -31,10 +62,24 @@ import (
 // Such operations should be wrapped in a timeout context.
 const reconfigTimeout = time.Second
 
+type response struct {
+	pkt  []byte
+	from netaddr.IPPort // where the packet needs to be sent
```

Comment:
Why is it `from`, if this is where the packet needs to be sent? The comment and code don't seem to match.

Reply:
To/From relativism strikes again XD Its the address where the request came from, I'll update the comment.